src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -241,6 +249,7 @@ struct sigaction       old_segv;
 /* the flag to indicate whether initlization is finished or not */
 static bool            hook_enabled;
 static bool            hook_enabled_bak;
+static pthread_mutex_t lock_global;
```

Comment:
not a big deal, but if a repush is needed, better to rename that to lock_reserve_fd

Reply:
Thank you! I will fix it if I need a repush.